README.md

```diff
@@ -110,7 +111,61 @@ def trainModel( embeddedData, dataWithScores ):
    return trainedModel
 ``` -->
 
-## Module 3: Results Selection and Evaluation
+## Module 2: Text Summarization
+### Function 2 retrieval
+* ***Functionality***: Apply the Dense Passage Retrieval (DPR) to evaluate and rank the relevance between the question and text, then select the most relevant ones as the original text
+* ***Input***:
+   * question: the question asked by the user
+   * raw_text: the raw text fetched from the Data Collection module 
+   * paragraph_num: the number of paragraphs user wants to apply for the original text
```

Comment:
Can you clarify what "number of paragraphs user wants to apply" means in the argument description for `paragraph_num`? Does this mean the number of paragraphs that are retrieved by the function?

Reply:
The paragraph_num means the number of paragraphs to compose the original text. I have changed the expression. Thank you.